src/optim/FishLeg/fishleg.py

```diff
@@ -285,11 +261,16 @@ def update_aux(self) -> None:
 
         aux_loss = 0
 
-        if self.method == "antithetic":
+        if method == "antithetic":
+            try:
+                eps = method_kwargs["eps"]
```

Comment:
eps = method_kwargs["eps"] if "eps" in method_kwargs else 1e-4

Reply:
this should work better. the try, except seems unnecessary to me here. Should be just an if condition